### DISCUSSION TITLE
Multiple Feed Queries

### DISCUSSION THREAD

**Discussion by melchionnaeu** [FreshRSS/FreshRSS#6534] [Q&A]
Can I use queries to filter multiple blogs and create a single rss?

Example:

Site 1: Query Filter 1
Site 2: Query Filter 2

Feed Rss of 1+2

**Comment by Alkarex** [comment]
Yes, you can create a single RSS output from multiple RSS inputs in a few different ways, such as:
* Put all the relevant feeds in the same category, and make an RSS feed of that category (using a user query)
* Make a search with a list of feed IDS of interest, such as `f:3,7,13`, save that user query, and share via RSS

See https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field
